assets/src/edit-story/components/canvas/karma/selection.karma.js

```diff
@@ -89,4 +89,19 @@ describe('Selection integration', () => {
     expect(await getSelection()).toEqual([element1.id]);
     await fixture.snapshot();
   });
+
+  it('should return focus to selection when pressing mod+alt+2', async () => {
+    // NB: We can't actually validate that the frame has focus, as that's a bit flaky,
+    // But as long as the focus moves in the shortcut press, it's fair to assume that it has
+    // Move to the canvas selection.
+
+    // Click elsewhere
+    await fixture.events.click(fixture.editor.canvas.header.title);
+    expect(fixture.editor.canvas.header.title).toHaveFocus();
+
+    // Return focus with shortcut
+    await fixture.events.keyboard.shortcut('mod+alt+2');
+    expect(fixture.editor.canvas.header.title).not.toHaveFocus();
+    await fixture.snapshot('selected element has focus');
```

Comment:
I know this isn't the _"strongest"_ test ever, but validating that focus is actually on the frame (which it eventually will be) is pretty flaky and works only _sometimes_. We could try to use arrow keys and validate, that the on-canvas object moves, which indicates that it has some kind of focus (though that's also a global listener, actually).

That's actually the key issue here. Most listeners on the selected element(s) are global listeners anyway, so it's not _that_ important that the specific element has focus - it's more important that the input field **doesn't**.